app/javascript/packs/MainApp/components/RefereeProfile.jsx

```diff
@@ -107,38 +152,41 @@ class RefereeProfile extends Component {
       )
     }
 
+    if (edit) {
+      return <RefereeProfileForm defaultValues={referee} onSubmit={this.save} />
+    }
+
     return (
-      <div>
-        <h1>
-          {`${firstName} ${lastName}`}
-        </h1>
+      <Fragment>
+        <Header as="h1">
+          {`${referee.firstName} ${referee.lastName}`}
```

Comment:
When a ref signs up they won't have a first or last name, if that's the case let's have some fallback text instructing refs to update their names.